src/style.css

```diff
@@ -0,0 +1,39 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("images/star.svg");
+  height: 16px;
+  width: 16px;
+  background-repeat: space;
```

Comment:
`space` works here but `no-repeat` is the exact value we want to use in this case.
Check [the docs](https://developer.mozilla.org/en-US/docs/Web/CSS/background-repeat)